### DISCUSSION TITLE
What does recruiter mainly see in Profile

### DISCUSSION THREAD

**Discussion by AyushJain001** [community/community#35037] [Other Feature Feedback, Questions, & Ideas]
I want to ask that what does recruiter see in GitHub profile for hiring. Please answer.

**Comment by radar-blip** [comment]
I would think the Contribution Graph and Activity Overview would be most important and not to misleading.

**Comment by vedanshtandon** [comment]
The tech stack used in your projects and the consistency which you have towards open source contribution
